src/libstd/io/util.rs

```diff
@@ -45,7 +45,8 @@ use io::{self, Read, Write, ErrorKind, BufRead};
 /// # }
 /// ```
 #[stable(feature = "rust1", since = "1.0.0")]
-pub fn copy<R: Read, W: Write>(reader: &mut R, writer: &mut W) -> io::Result<u64> {
+pub fn copy<R: ?Sized + Read, W: ?Sized + Write>(reader: &mut R, writer: &mut W)
+    -> io::Result<u64> {
```

Comment:
If you have time, what's our actual best style for wrapping function signatures?

libstd seems to want this:

A:
```
pub fn copy<R: ?Sized + Read, W: ?Sized + Write>(reader: &mut R,
                                                 writer: &mut W)
                                                 -> io::Result<u64> {
```

B:
```
pub fn copy<R: ?Sized, W: ?Sized>(reader: &mut R,
                                  writer: &mut W)
                                  -> io::Result<u64> where R: Read, W: Write {
```
I usually do this personally, dislike `{` hanging so far to the right

C:
```
pub fn copy<R: ?Sized, W: ?Sized>(reader: &mut R, writer: &mut W) -> io::Result<u64>
    where R: Read, W: Write
{
```